js/foundation.util.mediaQuery.js

```diff
@@ -109,11 +109,11 @@ var MediaQuery = {
       var newSize = this._getCurrentSize();
 
       if (newSize !== this.current) {
-        // Broadcast the media query change on the window
-        $(window).trigger('changed.zf.mediaquery', [newSize, this.current]);
-
         // Change the current media query
         this.current = newSize;
+
+        // Broadcast the media query change on the window
+        $(window).trigger('changed.zf.mediaquery', [newSize, this.current]);
```

Comment:
Triggering it here means this.current is newSize, rather than the old size.  I think moving this to after the set is reasonable, but if we do we should capture the previous value of this.current and pass it as the second value.